boilerplate/frontend/supertokens-react-custom/src/pages/Auth/emailpassword/useLogin.ts

```diff
@@ -0,0 +1,71 @@
+import { signIn } from "supertokens-web-js/recipe/emailpassword";
+import STGeneralError from "supertokens-web-js/utils/error";
+
+type LoginResponse = {
+    status: "success" | "failure";
+    reason: string;
+};
+function useLogin() {
```

Comment:
i dont think we need this type of pattern, cause we are calling these functions on button clicks anyway. So it has no benefit. Keep things as simple as possible throughout the boilerplate code.

Reply:
I understand your point that the hooks may not make sense as they are not holding any stateful values, hence I have moved them to `./utils..ts` files of the respective recipe. This ensures that the major business logics are in the utils file and also keeps the Line of Code less and manageable in the View/Component